wp/lib/bap_wp/src/environment.mli

```diff
@@ -251,6 +251,9 @@ val use_input_regs : t -> bool
     defined by the concrete range of the stack in the env. *)
 val in_stack : t -> Constr.z3_expr -> Constr.z3_expr
 
+(** [get_stack_bottom] retrieves the address of the bottom of the stack *)
```

Comment:
Can you add a bit of clarification that the stack is grows from top to bottom, and the stack bottom is not the same as its base?